rust/processor/src/processors/fungible_asset_processor.rs

```diff
@@ -227,6 +255,63 @@ fn insert_current_fungible_asset_balances_query(
     )
 }
 
+fn insert_current_unified_fungible_asset_balances_v1_query(
+    items_to_insert: Vec<CurrentUnifiedFungibleAssetBalance>,
+) -> (
+    impl QueryFragment<Pg> + diesel::query_builder::QueryId + Send,
+    Option<&'static str>,
+) {
+    use schema::current_unified_fungible_asset_balances::dsl::*;
+
+    (
+        diesel::insert_into(schema::current_unified_fungible_asset_balances::table)
+            .values(items_to_insert)
+            .on_conflict(storage_id)
+            .do_update()
+            .set(
+                (
+                    owner_address.eq(excluded(owner_address)),
+                    asset_type.eq(excluded(asset_type)),
+                    coin_type.eq(excluded(coin_type)),
+                    is_frozen.eq(excluded(is_frozen)),
+                    amount_v1.eq(excluded(amount_v1)),
+                    last_transaction_timestamp_v1.eq(excluded(last_transaction_timestamp_v1)),
+                    last_transaction_version_v1.eq(excluded(last_transaction_version_v1)),
+                    inserted_at.eq(excluded(inserted_at)),
+                )
+            ),
+        Some(" WHERE current_unified_fungible_asset_balances.last_transaction_version_v1 <= excluded.last_transaction_version_v1 "),
+    )
+}
+
+fn insert_current_unified_fungible_asset_balances_v2_query(
+    items_to_insert: Vec<CurrentUnifiedFungibleAssetBalance>,
+) -> (
+    impl QueryFragment<Pg> + diesel::query_builder::QueryId + Send,
+    Option<&'static str>,
+) {
+    use schema::current_unified_fungible_asset_balances::dsl::*;
+    (
+        diesel::insert_into(schema::current_unified_fungible_asset_balances::table)
+            .values(items_to_insert)
+            .on_conflict(storage_id)
+            .do_update()
+            .set(
+                (
+                    owner_address.eq(excluded(owner_address)),
```

Comment:
If there is a conflict between v1 and v2, are these (owner_address, asset_type, is_frozen) fields the same?

Reply:
wdym?

Reply:
The the order of conflict upserts matters?

Reply:
I think maybe we should ignore this field for v1.